src/GovUk.Education.ExploreEducationStatistics.Data.Api/Startup.cs

```diff
@@ -198,7 +198,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             CancellationTokenTimeoutAspect.Enabled = true;
             CancellationTokenTimeoutAttribute.SetTimeoutConfiguration(Configuration.GetSection("RequestTimeouts"));
 
-            UpdateDatabase(app);
+            // TODO EES-3369 - We don't need to run these migrations against the statistics database in local
+            // development, as the Admin project will have applied these migrations already against the sole statistics
+            // database but in Azure environments, the Data API is responsible still for applying migrations to 
+            // the `public-statistics` database.
+            if (!env.IsDevelopment())
```

Comment:
As part of Stage 2 of the read replica rollout, when it's only `adminapp` making changes to the PRIMARY statistics db, we can remove the need to apply any migrations from the Data API project entirely.

For now though, we'll still need to allow the `data` user to apply migrations for the `public-statistics` database.

Reply:
I think we could have an issue in future where the window of time between the Admin app starting up after a deploy and running the `statistics` migrations, and the old version of the Data API being take down to begin a deploy, is quite large.

It's especially large right now with the slot swapping timeout shenanigans (EES-3211) and content API deploys before the Data API.

There could be lots of errors during that time.

I guess we already have a similar problem now with function apps and the app services that deploy after Admin all sharing the content database which is migrated by the Admin while they remain up.

Reply:
Yeah, and maybe we need to start thinking about the dreaded "expand / contract" pattern for breaking changes so that we don't cause disruption in the future - one to bring up in devs meeting?

Reply:
Thanks, I've added it to the dev's team meeting board to discuss.